src/atlas/util/VortexRollup.cc

```diff
@@ -0,0 +1,43 @@
+/*
+ * (C) Crown Copyright 2022 Met Office
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation
+ * nor does it submit to any jurisdiction.
+ */
+
+#include "atlas/util/VortexRollup.h"
+
+namespace atlas {
+
+namespace util {
+
+double vortex_rollup(double lon, double lat, double t,
+                     const double mean, const bool degreesInput,
+                     const double timescale) {
+
+  if (degreesInput) {
+    lon *= Constants::degreesToRadians();
+    lat *= Constants::degreesToRadians();
+  }
+
+  auto sqr           = [](const double x) { return x * x; };
+  auto sech          = [](const double x) { return 1. / std::cosh(x); };
+  const double Omega = 2. * M_PI / timescale;
```

Comment:
```
constexpr double two_pi = 2. * M_PI;
```
Given that the timescale is meaningless (see other comments), we could just keep using `two_pi` instead of `Omega`

Reply:
Done